projects/graphql/eslint-plugin-graphql/parser.js

```diff
@@ -72,6 +73,15 @@ module.exports.parseForESLint = function (code, options = {}) {
         return this.correspondingNode(ast);
       },
 
+      /**
+       * This exists for rules to get the same version of the graphql library
+       * used so that `Duplicate "graphql" modules cannot be used at the ...`
+       * error can be avoided
+       */
+      getGraphQL() {
+        return graphql;
+      },
+
       createTypeInfo() {
         return new TypeInfo(schema);
       },
```

Comment:
If you have time to add docs to https://github.com/stefanpenner/eslint-ast/blob/master/projects/graphql/eslint-plugin-graphql/README.md that would be great, but either way i'm happy to merge + release.

Reply:
Added documentation for the functions provided by the parser service